testcase/deployment.go

```diff
@@ -18,10 +18,6 @@ type Deployment struct {
 	nginx2Deployment      *v1.Deployment
 	nginx3Deployment      *v1.Deployment
 	timeout               time.Duration
-	serviceAccountName    string
```

Comment:
Why do we need remove `serviceAccountName`

Reply:
We do not need to create serviceAccount.